RetailCoder.VBE/UI/Refactorings/ReorderParameters/ReorderParametersDialog.resx

```diff
@@ -0,0 +1,142 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@ckuhn203  resx is the WinForms format for resource files.... FYI WPF localization ditches the .resx format :wink: